Allow for avatar and toggle customization (#335)
patternfly/chatbot: packages/module/patternfly-docs/content/extensions/chatbot/examples/Messages/Messages.md

By default, a date and timestamp is displayed with each message. We recommend using the `timestamp` prop in real chatbots, since it will allow you to set persistent dates and times on messages, even if the messages re-render. You can update `timestamp` with a different [date and time format](/ux-writing/numerics) as needed.

<br />

> If you need to adjust the avatar further, you can also apply `avatarProps` to the Message to adjust any [PatternFly Avatar](/components/avatar/) props you might need to change, or to apply an additional class.

Reviewer: ```suggestion
You can further customize the avatar by applying an additional class or passing [PatternFly avatar props](/components/avatar) to the `<Message>` component via `avatarProps`.
```
just mirroring my previous suggestion

### User messages

> Messages from users have a different background color to differentiate them from bot messages. You can also display a custom avatar that is uploaded by the user. You also apply `avatarProps` to the Message to adjust any [PatternFly Avatar](/components/avatar/) props you might need to change, or to apply an additional class.

Reviewer: ```suggestion
Messages from users have a different background color to differentiate them from bot messages. You can also display a custom avatar that is uploaded by the user. You can further customize the avatar by applying an additional class or passing [PatternFly avatar props](/components/avatar) to the `<Message>` component via `avatarProps`.
```
